docs/logs/customizing-the-sdk/README.md

```diff
@@ -32,7 +32,13 @@ set by invoking the formatter from [ILogger.Log](https://docs.microsoft.com/dotn
 
 ### ParseStateValues
 
-TODO
+ILogger supports [message templates](https://docs.microsoft.com/dotnet/core/extensions/logging#log-message-template)
```

Comment:
this is irrelevant.
check the code doc for `ParseStateValues`, and copy what we have there.

Reply:
The code doc states: "Gets or sets a value indicating whether or not log state should be parsed into LogRecord.StateValues on generated LogRecords."

I don't think this does a good job of explaining what the log state is or what data will be set into LogRecord.StateValues.

Here, I was trying to introduce what the log state is and what parts of it are set in the StateValues.